pages/content/amp-dev/documentation/guides-and-tutorials/start/visual_story/setting_up.md

```diff
@@ -11,7 +11,7 @@ author: bpaduch
 Before starting this tutorial, you'll need the following:
 
 - A basic knowledge of HTML, CSS, and JavaScript
-- A basic understanding of AMP’s core concepts (see ["Convert your HTML to AMP"]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/converting/index.md', locale=doc.locale).url.path}}) tutorial)
+- A basic understanding of AMP’s core concepts (see ["Convert your HTML to AMP"](https://amp.dev/documentation/guides-and-tutorials/start/converting/?format=websites) tutorial)
```

Comment:
They all look good expect for this one! :)

Reply:
done. thanks